packages/elements/src/photon-authorized/index.tsx

```diff
@@ -0,0 +1,51 @@
+//Shoelace
+import '@shoelace-style/shoelace/dist/components/dialog/dialog';
+import { setBasePath } from '@shoelace-style/shoelace/dist/utilities/base-path.js';
+
+setBasePath('https://cdn.jsdelivr.net/npm/@shoelace-style/shoelace@2.0.0-beta.82/dist/');
+
+import { createEffect, createSignal, JSXElement, Show } from 'solid-js';
+import { usePhoton } from '../context';
+
+export const PhotonAuthorized = (props: { children: JSXElement }) => {
+  const client = usePhoton();
+  const [isLoading, setIsLoading] = createSignal<boolean>(
+    client?.authentication.state.isLoading || false
+  );
+  const [authenticated, setAuthenticated] = createSignal<boolean>(
+    client?.authentication.state.isAuthenticated || false
+  );
+  const [authorized, setAuthorized] = createSignal<boolean>(
+    client?.authentication.state.isAuthorized || false
+  );
+
+  createEffect(() => {
+    setIsLoading(client?.authentication.state.isLoading || false);
+  });
+  createEffect(() => {
+    setAuthenticated(client?.authentication.state.isAuthenticated || false);
+  });
+  createEffect(() => {
+    setAuthorized(client?.authentication.state.isAuthorized || false);
+  });
+
+  return (
+    <>
+      <Show when={client && !authenticated()}>
```

Comment:
@samkotlove this should handle the not authenticated message you mentioned below

<img width="709" alt="Screen Shot 2023-02-20 at 12 33 38 PM" src="https://user-images.githubusercontent.com/700617/220170692-d1909f70-ad95-4598-b2b2-25f9a9f6904f.png">